### ISSUE TITLE
selenium-server manager feature

### ISSUE THREAD

**Issue by udarrr** [webdriverio/selenium-standalone#828]
Hello @christian-bromann  What do you think regarding the feature https://www.selenium.dev/blog/2023/whats-new-in-selenium-manager-with-selenium-4.11.0/ ?

**Comment by christian-bromann** [comment]
The Selenium manager project is great. Are you asking about a specific feature?

**Comment by udarrr** [comment]
> The Selenium manager project is great. Are you asking about a specific feature?

No, i meant in general. Maybe have sense implement functionality of the package in selenium-standalone tool ? :)

**Comment by christian-bromann** [comment]
> Maybe have sense implement functionality of the package in selenium-standalone tool ? :)

I am totally okay with that. If it helps remove some of the code here it makes sense. Let's just ensure we don't break any functionality if possible.

**Comment by udarrr** [comment]
>Let's just ensure we don't break any functionality if possible.

Quite complicated to do so with just pure js :) I think before big changes it should be rewritten to ts. Maybe i'll look into it
